apps/admin/frontend/src/screens/election_screen.tsx

```diff
@@ -61,23 +61,17 @@ export function ElectionScreen(): JSX.Element {
           </P>
         </div>
       </ElectionCard>
-      {isSystemAdministratorAuth(auth) && (
-        <UnconfigureMachineButton
-          isMachineConfigured
-          unconfigureMachine={unconfigureMachine}
-        />
-      )}
-      {isElectionManagerAuth(auth) && (
-        <React.Fragment>
-          <P>
-            Save the election package to the USB drive to configure VxSuite
-            components.
-          </P>
-          <P>
-            <ExportElectionPackageModalButton />
-          </P>
```

Comment:
Any reason why we would have wrapped the Button in a `P` previously?

Reply:
It's a legacy pattern, feel free to remove. The only reason to keep it would be if it's needed for spacing